ui/src/cohortReview/reviewHooks.ts

```diff
@@ -21,10 +21,11 @@ export type ReviewParams = {
 };
 
 export function useReviewParams(): ReviewParams {
+  const underlaySource = useUnderlaySource();
   const underlay = useUnderlay();
   const params = useBaseParams();
 
-  const primaryKey = underlay.uiConfiguration.dataConfig.primaryEntity.key;
+  const primaryKey = underlaySource.lookupEntity("").idAttribute;
```

Comment:
nit: Consider an overloaded function e.g. `underlaySource.primaryEntity()` instead of the special empty string arg.

Reply:
Done.